File: C#/1252. Cells with Odd Values in a Matrix.cs
public class Solution {
    public int OddCells(int m, int n, int[][] indices) {
		int[,] arr = new int[m,n];
		int res = 0;
		for(int i=0; i<indices.Length; i++){
			for(int j=0; j<m; j++){
				arr[j,indices[i][1]]++;
			}
			for(int k=0; k<n; k++){
				arr[indices[i][0],k]++;
			}
		}
		for(int i=0; i<m; i++){
			for(int j=0; j<n; j++) 
				if(arr[i,j]%2==1){
					res++;
				}
		}
		return res;
    }
}

File: C++/1252. Cells with Odd Values in a Matrix.cpp
class Solution {
public:
    int oddCells(int m, int n, vector<vector<int>>& indices) {
		vector<vector<int>> vec(m,vector<int>(n));
		int res = 0;
		for(int i=0; i<indices.size(); i++){
			for(int j=0; j<m; j++){
				vec[j][indices[i][1]]++;
			}
			for(int k=0; k<n; k++){
				vec[indices[i][0]][k]++;
			}
		}
		for(int i=0; i<m; i++){
			for(int j=0; j<n; j++) 
				if(vec[i][j]%2==1){
					res++;
				}
		}
		return res;
    }
};
